src/plugins/navigation/public/plugin.tsx

```diff
@@ -129,39 +127,65 @@ export class NavigationPublicPlugin
 
     const config = this.initializerContext.config.get();
     const {
-      solutionNavigation: { featureOn: isSolutionNavigationFeatureOn },
+      solutionNavigation: { defaultSolution },
     } = config;
 
     const onCloud = cloud !== undefined; // The new side nav will initially only be available to cloud users
     const isServerless = this.initializerContext.env.packageInfo.buildFlavor === 'serverless';
-    const isSolutionNavEnabled = isSolutionNavigationFeatureOn && onCloud && !isServerless;
-    this.isSolutionNavEnabled$ = of(isSolutionNavEnabled);
 
-    if (isSolutionNavEnabled) {
-      chrome.project.setCloudUrls(cloud);
-      this.addDefaultSolutionNavigation({ chrome });
+    let isFeatureEnabled$ = of(false);
```

Comment:
It is not clear what is the feature on this point, I'd call it isSolutionNavExperiementEnabled$

Reply:
Addressed in https://github.com/elastic/kibana/pull/179913/commits/cc93dd7af1d7ef0bbf78656e2459157acdc4b65e 👍